engine/src/flutter/lib/web_ui/lib/src/engine/renderer.dart

```diff
@@ -35,6 +35,7 @@ abstract class Renderer {
     }
   }
 
+  Arena get frameArena;
```

Comment:
The arena itself seems to be renderer-agnostic. Should we put it in `EnginePlatformDispatcher` instead?